hub/cli/auth.py

```diff
@@ -38,6 +49,19 @@ def logout():
     click.echo("Logged out of Hub.")
 
 
+@click.command()
+@click.option("--on/--off", help="Turn crash report on/off")
+def reporting(on):
+    """Enable or disable sending crash report to Activeloop AI"""
```

Comment:
Please add a TODO to add this to our docs
`TODO: Add how to enable/disable reporting to docs`